litmus-portal/frontend/src/pages/CreateCustomWorkflow/index.tsx

```diff
@@ -0,0 +1,45 @@
+import React from 'react';
+import Scaffold from '../../containers/layouts/Scaffold';
+import CreateWorkflow from '../../views/CreateWorkflow/CustomWorkflow/CreateWorkflow';
+import ScheduleCustomWorkflow from '../../views/CreateWorkflow/CustomWorkflow/ScheduleWorkflow';
+import TuneCustomWorkflow from '../../views/CreateWorkflow/CustomWorkflow/TuneWorkflow';
+import ExperimentEditor from '../../views/CreateWorkflow/CustomWorkflow/YamlEditor';
+import useStyles from './styles';
+
+function getStepContent(
+  stepIndex: number,
+  gotoStep: (page: number) => void
+): React.ReactNode {
+  switch (stepIndex) {
+    case 0:
+      return <CreateWorkflow gotoStep={(page: number) => gotoStep(page)} />;
+    case 1:
+      return <TuneCustomWorkflow gotoStep={(page: number) => gotoStep(page)} />;
+    case 2:
+      return (
+        <ScheduleCustomWorkflow gotoStep={(page: number) => gotoStep(page)} />
+      );
```

Comment:
```suggestion
      return <ScheduleCustomWorkflow gotoStep={(page: number) => gotoStep(page)} />;
```

Reply:
@ishangupta-ds  tried the suggested change but I think the changes are getting reverted after linting.